app/helpers/application_helper.rb

```diff
@@ -79,10 +79,6 @@ def revision_path(rev)
     notebook_revision_path(rev.notebook, rev)
   end
 
-  def link_to_revision(rev)
```

Comment:
For backwards compatibility, I think this should stay but at most throw a rails warning about deprecation.

Reply:
That makes sense. Will do